beacon-chain/db/kv/altair.go

```diff
@@ -0,0 +1,14 @@
+package kv
+
+import "bytes"
+
+// In order for an encoding to be Altair compatible, it must be prefixed with altair key.
+func hasAltairKey(enc []byte) bool {
+	if len(altairKey) >= len(enc) {
+		return false
+	}
+	if bytes.Equal(enc[:len(altairKey)], altairKey) {
```

Comment:
You can simplify further
```suggestion
	return bytes.Equal(enc[:len(altairKey)], altairKey) 
```